### ISSUE TITLE
Feedback on the recent script changes

### ISSUE THREAD

**Issue by Commenter25** [Foohy/jazztronauts#149]
Recently, @wertercatt made a huge PR fixing many typos. Which is fantastic, and a lot of effort and initiative, which I respect and appreciate! I had never thought to before, probably because I wasn't technically inclined enough when I played through in the past. 

However, I feel far too many stylistic and grammar changes were made, which lost some of the casual and informal tone that made these characters feel more real. And I'm not sure the PR was thoroughly reviewed to catch these things. Though honestly, it may be me overanalyzing and being way too pedantic. Actually, it probably is. But I thought I should bring this up in case I'm not crazy. It's all subjective, and different interpretations of language and writing, though it's something I care about enough to mention.

Changing all the names from "the Title" to "The Title" doesn't seem right. It isn't part of their title, their names are simply the one word. We don't write "The Grand Canyon" on maps, we write "Grand Canyon", so we call it "the Grand Canyon". So if we can just call the Pianist "Pianist", why write "The Pianist" instead of "the Pianist"?

There are also lots of line changes like "Have you seen the Cellist about?" to "Have you seen the Cellist around?", or "You got The Goods yet?" to "Did you get The Goods yet?", or even just going from "this'll" to "this will". All of which are more *technically* correct, but in my opinion, technical correctness in English is worthless. Conveying meaning, tone, and character is more important; and the former statements feel far more loose. Like something a real person would say.

While this is FAR more subjective, I can say the same about a lot of changes to comma placement. There were a lot of commas changed in a grammatically correct way, but in my opinion, it changed how sentences are read. I feel like strict correctness holds the comma's power back, and I think it's best used as an audible pause, and breaking a long sentence for readability. You can see what I mean about audible pauses by thinking of how you read "At first, this happened and" versus reading "At first this happened, and"

Another example of this comma placement thing, a line changed in this PR: "You feel a sudden lurch in your stomach, and the whole world breaks like a cheap wine glass!" The comma in the middle was removed. The sentence isn't wrong now, but now it feels much more long winded, because there's no pause in the middle.

I also wanna mention everything was switched from British to American, which just seems unnecessary. Though I did notice "color" is used exclusively outside of variable names, so I'm not quite sure what's preferred. But it should be consistent, and ideally whatever Foohy would typically write, or would associate these characters using.

I think that's the major stuff. I'm not putting this all out and expecting someone else to do it, I'll open a PR based on how my thoughts are received. Again, this is all subjective, and if I'm wrong, that's okay! I put this as an issue first to spark discussion, and hopefully get the thoughts of @Foohy; since ultimately, I am not the author, and authorial intent comes before all else here. And again, absolutely no hate to you @wertercatt and I hope to not offend, it was a huge PR which made plenty of changes I do agree with, and you've done even more for Jazztronauts since! Above all, I want this game to be the best it can be.

**Comment by ZakBlystone** [comment]
Thanks for bringing this up. Foohy and I had a good long talk about this and we agree with the points you have made. However, we don't really want to back out the entire PR because:
A. there were some changes that made sense and are beneficial to the project
B. it probably took a lot of work to make, and we'd feel pretty bad about throwing it out

In order to address these issues, we would need to manually sift through and separate the wheat from the chaff, but we simply just don't have that kind of free time at the moment. We are both very busy at our respective jobs and cannot commit effort to that kind of task right now.

However, if you are feeling up to the task, you could submit your own Pull Request to revert the particular changes you feel were made in error, and help us strike a balance between the changes that were made and the original style of Jazztronauts.

If you decide not to, that's fine... it just means it will probably be a while before this gets addressed.

Thanks,
-Zak

**Comment by Commenter25** [comment]
Is `black_shard_endings redo.txt` serving any purpose? It seems to be long outdated. Should it be removed?

**Comment by wertercatt** [comment]
Hi, I mainly just combed through it all with Grammarly. Feel free to comb through it yourself to revert the changes you disagree with, but I can confirm that reading the scripts is quite hard with all the in-lined animations and stuff. I'm not particularly interested in digging through all of them again so soon, but I wish you the best of luck in your own endeavors with them.

**Comment by wertercatt** [comment]
Going to add some tips for you @Commenter25:

I'm pretty sure `black_shard_endings redo.txt` is a leftover dialogue script used when they were testing the rewritten ending. I didn't remove it because I hadn't wrapped my head around how the game loads scripts yet. I think I have now, and there's nothing that uses it as far as I can tell, but your mileage may vary if you remove it.

Take your time and work through each script file as its own commit, and probably take breaks between them. Staring at it too long can be incredibly tedious. You may want to make individual PRs at different milestones so they're easier to be reviewed.

As for `the Cellist` vs `The Cellist`, any change made to standardize it will inherently be subjective. He was referred to as both, as well as just `Cellist` randomly throughout the dialogue. I standardized on `The Cellist` because it seemed better to me, but you're welcome to change it to the other one.

You can use `jazz_debug_runscript` in the console with the filename of a script to play it from the `begin:` marker. I didn't discover this until after my big fix PR, but it should be helpful to test your changes in-game without having to replay sections.

**Comment by Commenter25** [comment]
Appreciate the advice!

I figured out redo was unused yeah, that's why I asked about it. Like I mentioned, it seems outdated, having less commit history and a lot less polish. I just don't want to remove it without asking, since maybe having it there is desired for whatever reason.

Yeah, I agree. I was initially going to make a big PR, but I realized that's going to be a rather daunting PR to look at. If I separate them, they get through faster, they're easier to review, anyone else who might try to contribute won't get confused, there hopefully won't be any merge conflicts... it just seems overall better. I've already done the intro (though I still want to skim through a few times to be sure, and if #152 is merged I'll utilize that too), then I plan on doing each character's missions/events (so a Cellist PR, then Singer PR, etc), and then I'll do all the endings stuff. I may even just put up a draft PR for each section, which will get updated with a commit for each file I finish.

To my knowledge, it's standard to not capitalize "the" before a proper noun unless it's an integral part of the name. Since you can just call him "Cellist", that's his whole name, and you don't capitalize "the", or else his full name would be "The Cellist". I'll admit it's rather nitpicky of me, but it just rubs me wrong. Either way, it comes off more formal to lean on capitalizing it, and I just don't like being formal :p I've actually got a full commit ready to make it all the same, but I'm waiting to see whether or not #153 gets merged, because if I publish it now there might be messy merge conflict stuff and then it couldn't be merged anyway. Though it should be quick to put out whichever way that goes. And once the capitalization thing is merged, I shouldn't have to wait before putting anything out.

Thankfully I figured that command out pretty quick, since I went through all the `jazz_` commands to see what would help out. I still restarted the intro far more times than I would've liked, but once I figured that out it's been smooth sailing.
